modules/tide_paragraphs_enhanced_modal/css/paragraphs.enhanced_modal.css

```diff
@@ -33,6 +33,7 @@
   box-shadow: none !important;
   border: none !important;
   padding: 0 0 0.5em;
+  text-wrap: auto;
```

Comment:
@yeniatencio auto does not actually count as a valid value anymore and it is not a global value as well. In this case you should use a valid value for text-wrap it self. In this case use text-wrap: wrap; or text-wrap: balance; or text-wrap: pretty; or text-wrap: stable;

```
/* Keyword values */
text-wrap: wrap;
text-wrap: nowrap;
text-wrap: balance;
text-wrap: pretty;
text-wrap: stable;

/* Global values */
text-wrap: inherit;
text-wrap: initial;
text-wrap: revert;
text-wrap: revert-layer;
text-wrap: unset;
```

Reply:
Thank you @MdNadimHossain . I have updated my PR.